cropper/src/main/java/com/canhub/cropper/CropImageOptions.kt

```diff
@@ -96,6 +96,10 @@ open class CropImageOptions : Parcelable {
     @JvmField
     var showProgressBar: Boolean
 
+    /** The color of the progress bar. Only works on API level 21 and upwards. */
```

Comment:
Would be great to add a suggestion about how to use this. a.k.a. `Color.rgb(153, 51, 153)` 

This is because people can send the wrong Int, like `R.color.LoadingColor` which would be wrong, since is a `ResInt` value.

Or even have both options, `int` and `resInt` so is easy for the user too choose.

Reply:
Added annotations on all colors. I'd leave out the Color.rgb. you could also use `Color.RED` or `resources.getColor`, we should not explain android development to our library users